internal/merger/merger.go

```diff
@@ -366,3 +366,43 @@ func toStatVarSummaryMap(in []*pbv1.VariableInfoResponse) map[string]*proto.Stat
 	}
 	return out
 }
+
+// MergeSearchStatVarResponse merges two SearchStatVarResponse.
+func MergeSearchStatVarResponse(primary, secondary *proto.SearchStatVarResponse) *proto.SearchStatVarResponse {
+	mergedStatVars := []*proto.EntityInfo{}
+	dedupedMatches := []string{}
+	matchesMap  := map[string]bool{}
```

Comment:
The go convention for a set-like map is `map[string]struct{}` ([example](https://github.com/datacommonsorg/mixer/blob/08a8e50c6c5c935159ea8e4a6986408d1cb4c66f/internal/util/util.go#L720)). Consider using that.

Reply:
Done.